### DISCUSSION TITLE
Since iina is based on mpv, what would I have to do so I can use the mpv remote app on Android, if possible?

### DISCUSSION THREAD

**Discussion by cookies92** [iina/iina#4349] [Q&A]
I looked through the settings in iina, and I looked in some documentation, but I didn't quite find what I was looking for.

**Comment by low-batt** [comment]
You are correct, a lot of mpv functionality is available in IINA. To configure mpv options and enable use of mpv config files, follow these instructions:
- Start IINA
- Click on  `Settings…` under the `IINA` menu
- The settings panel appears
- On the left side of the panel click on `Advanced`
- Slide the `Enable advanced settings` toggle button to be on (blue)
- Check the `Use config directory` setting near the bottom of the panel
- In the `Additional mpv options` section click on `+`
- A new entry appears in the table
- Double click on `name` and replace it with the mpv option you want to set
- Double click on `value` and replace it with the value for that option
- **Restart IINA** to activate the new settings

I assume this is about the open source project [mpv-remote-app](https://github.com/husudosu/mpv-remote-app) and its associated [mpv-remote-node](https://github.com/husudosu/mpv-remote-node). Have you looked into the security aspects of running a server on your Mac? Proceed with caution.
